time stamp
int8 alert_robot
int8 path_num
string catch
